### PR TITLE
Update prototyping-utilities.md

### PR DESCRIPTION
**Pull request by chrisdfe**
## Description

Removed an unnecessary apostrophe in the description there.

## Types of changes

- [x] Documentation
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (anything that would change an existing functionality)
- [ ] Maintenance (refactor, code cleaning, development tools...)

## Checklist

- [x] I have read and follow the CONTRIBUTING.md document.
- [x] The pull request title and template are correctly filled.
- [x] The pull request targets the right branch (`develop` or `develop-v...`).
- [x] My commits are correctly titled and contain all relevant information.
- [ ] I have updated the documentation accordingly to my changes (if relevant).
- [ ] I have added tests to cover my changes (if relevant).

### CODE REVIEW

**Review by ncoden — COMMENTED** [review]
Hi @chrisdfe 👋,

I have an idea about what the whole sentence should mean, but "into hi-fi coded prototypes ultra fast." look like words were put in the wrong order.

I am missing something? I'm not a native english speaker.

**Comment by chrisdfe** [comment]
Hi @ncoden!
I see what you mean - that whole sentence is a little confusing. The apostrophe in "prototypes" was definitely incorrect grammar though.

Do you think this reads any better?
"These optional classes and mixins are great for quickly turning sketches and mockups into hi-fi coded prototypes."

**Comment by ncoden** [comment]
@chrisdfe Yes this is way better, thank you! Could you provide a commit for this?

**Comment by chrisdfe** [comment]
👍

File: docs/pages/prototyping-utilities.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Prototyping Utilities
-description: Quickly prototype layouts and UI with Foundation's Prototyping Utilities. These optional classes and mixins are great realizing your sketches and mockups into hi-fi coded prototype's ultra fast.
+description: Quickly prototype layouts and UI with Foundation's Prototyping Utilities. These optional classes and mixins are great for quickly turning sketches and mockups into hi-fi coded prototypes.
```
Comment (SassNinja) [inline docs/pages/prototyping-utilities.md]: I'm not an English native speaker either but the `hi-fi` sounds really weird to me.
I would just write `and mockups into coded prototypes.`

**Review by ncoden — APPROVED** [review]

**Pull request merged**